docs/sections/contributor_guide/developer_setup.rst

```diff
@@ -1,5 +1,183 @@
+these tests during development and, crucially, before merging changes,
+to ensure that the tests will pass when CI builds the ``workflow-tools``
+code.
+
+
+The order of the targets above is intentional, and possibly useful:
+
+   * ``make format`` will complain about certain kinds of syntax errors that
+     would cause all the remaining code-quality tools to fail (and may change
+     line numbers reported by other tools, if it ran after them).
+   * ``make lint`` provides a good first check for obvious errors and
+     anti-patterns in the code.
+   * ``make typecheck`` offers a more nuanced look at interfaces between 
+     functions, methods, etc. and may spot issues that would cause 
+     ``make unittest`` to fail.
+
+
+In addition to the ``make devshell`` command, other ``make`` targets
+are available for use *outside* a development shell, i.e. from the base
+conda environment (requires presence of the ``condev`` package):
+
+
++------------------+-------------------------------------------------------+
+| Command          | Description                                           |
++==================+=======================================================+
+| ``make env``     | Creates a conda environment based on the ``uwtools``  |
+|                  | code                                                  |
++------------------+-------------------------------------------------------+
+| ``make meta``    | Update ``recipe/meta.json`` from ``recipe/meta.yaml`` |
++------------------+-------------------------------------------------------+
+| ``make package`` | Builds a ``uwtools`` conda package                    |
++------------------+-------------------------------------------------------+
+
+
+These targets work from the code in its current state in the clone.
+``make env`` calls ``make package`` automatically to create a local
+package, then builds an environment based on the package.
+
+Building condev locally
+-----------------------
+
+As an alternative to installing a prebuilt package from
+`anaconda`_, the ``condev`` package can be
+built locally, then installed into the local conda installation. 
+Ensure that ``conda-build`` and ``conda-verify`` are installed in the base
+environment:
+
+   .. code:: sh
+
+     # Activate your conda
+     git clone https://github.com/maddenp/condev.git
+     make -C condev package
+     conda install -y -c local condev
+
+Files derived from condev
+-------------------------
+
+The following files in this repo are derived from their counterparts in
+the `condev demo`_:
+
+
+   .. code:: sh
+
+     ├── Makefile
+     ├── recipe
+     │   ├── build.sh
+     │   ├── channels
+     │   ├── meta.json
+     │   ├── meta.yaml
+     │   └── run_test.sh
+     ├── src
+     │   ├── pyproject.toml
+     │   ├── setup.py
+
+Behaviors described in previous sections may rely on these files
+continuing to follow ``condev`` conventions.
+
+
+.. |black| replace:: ``black``
+.. |isort| replace:: ``isort``
```

Comment:
Although this replacement/formatting is specific to the current `Developer Setup` section, it can be reusable and placed inside the `external_links` rst. For now I left it here, but that does leave the question open to renaming `external_links` to something more general.

Reply:
Good question -- we'd probably want to factor it out to the shared file if we need to repeat it on any other pages, but maybe we should just keep that possibility in mind and wait to see if it happens. I do like having all the links themselves in one place, though, as it makes it clear where new links should be placed.

Reply:
This has also now become obsolete with the change to make the packages hyperlinks instead. I thought this was the best decision since even if we did have the links a bit hidden by the fact it appeared in code, there would be almost no point to include them at all besides if a user viewing the documentation happened across them.